src/implementations/twig/components/fact-figures/fact-figures.story.js

```diff
@@ -1,135 +1,117 @@
 import { withNotes } from '@ecl/storybook-addon-notes';
-import {
-  withKnobs,
-  boolean,
-  text,
-  button,
-  select,
-} from '@storybook/addon-knobs';
 import withCode from '@ecl/storybook-addon-code';
-import { getExtraKnobs, tabLabels } from '@ecl/story-utils';
+import { correctSvgPath, getOptions } from '@ecl/story-utils';
 
 import generalIcons from '@ecl/resources-ec-icons/dist/lists/general.json';
-import defaultSprite from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
 import data3Col from '@ecl/specs-component-fact-figures/demo/data--3-col';
 import data4Col from '@ecl/specs-component-fact-figures/demo/data--4-col';
 
 import factFigures from './fact-figures.html.twig';
 import notes from './README.md';
 
-const iconsList = [];
-generalIcons.forEach((icon) => {
-  iconsList.push(icon);
-});
+const viewAll = { ...data3Col.view_all };
 
-// Preserve original data.
-const items3 = JSON.parse(JSON.stringify(data3Col.items));
-const items4 = JSON.parse(JSON.stringify(data4Col.items));
-const data3 = { ...data3Col };
-const data4 = { ...data4Col };
-// Button callback.
-// 3 Columns.
-const viewAll3Toggler = () => {
-  data3.view_all = data3.view_all ? false : data3Col.view_all;
+const getArgTypes = (data) => {
+  return {
+    icon: {
+      defaultValue: data.items[0].icon.name,
+      description: 'Name of the icon',
+      control: {
+        type: 'select',
+        options: getOptions(generalIcons),
+      },
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content (first item)',
+      },
+    },
+    value: {
+      type: { name: 'string', required: true },
+      defaultValue: data.items[0].value,
+      description: 'Main heading',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content (first item)',
+      },
+    },
+    title: {
+      type: { name: 'string', required: true },
+      defaultValue: data.items[0].title,
+      description: 'Sub heading',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content (first item)',
+      },
+    },
+    description: {
+      type: { name: 'string' },
+      defaultValue: data.items[0].description,
+      description: 'Description',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content (first item)',
+      },
+    },
+    viewAll: {
+      name: 'view all link',
+      type: { name: 'boolean' },
+      defaultValue: true,
+      description: 'Link in the component footer',
+      table: {
+        type: { summary: 'object' },
+        defaultValue: { summary: '{}' },
+        category: 'Others',
+      },
+    },
+    displayIcons: {
+      name: 'icons visibility',
+      type: { name: 'boolean' },
+      defaultValue: true,
+      description: 'Toggle visibility of the icons',
```

Comment:
Checking https://v3--europa-component-library.netlify.app/playground/ec/?path=/story/components-fact-figures--columns-3 the toggle is related to the descriptions, not the icons. Not an issue obviously, just a difference without notes in the pull request

Reply:
Another difference: no control for view_all.link.label

Reply:
Yeah, but we are following the ECL v2 knobs, not the ecl-twig ones although i actually added some here, to fully experience an item customisation (only one). 
The description can be reset by simply removing the text inside the control, so that's why i haven't been adding a "toggle" for it, the ones i placed are the ones from V2, basically.
Since there is the toggle i believe is not worth to place a control for the label of the link, otherwise i would be better to remove the toggle and delete the object when the control is reset, but i would keep it as it is.

![image](https://user-images.githubusercontent.com/14326885/103521137-ee8e0f80-4e80-11eb-85bb-f3d8af14f502.png)